File: Tests/src/com/jimcorp/tests/FizzBuzz.java
package com.jimcorp.tests;

public class FizzBuzz {

	public static void main(String[] args) {
		
		Integer nums = 100;
		
		for(Integer i=1; i<=nums; i++) {
			String message = "";
			if((i % 3) == 0) {
				message = "Fizz";
			}
			if((i%5) == 0) {
				message = message + "Buzz";
			}
			if(message.isEmpty()) {
				message = i.toString();
			}
			
			System.out.println(message);
		}

	}

}

File: PythonTests/src/FizzBuzz.py

nums = range(1,101)

for num in nums:
    message = ""
    if (num%3)==0:
        message = "Fizz"
    if (num%5)==0:
        message = message + "Buzz"
    if(len(message) == 0):
        message = num
    
    print(message)